studyzen/Set/FlashCardSetService.cs

```diff
@@ -0,0 +1,84 @@
+using StudyZen.FlashCards.Requests;
+using StudyZen.Persistence;
+using StudyZen.FlashCardSetClass;
+
+
+namespace StudyZen.FlashCardSets
+{
+
+    public interface IFlashCardSetService
+    {
+        FlashCardSet AddFlashCardSet(CreateFlashCardSetRequest request); 
+        FlashCardSet GetFlashCardSet(int flashCardSetId);
+        public bool DeleteFlashCardSet(int flashCardSetId);
+        public void UpdateFlashCardSet(FlashCardSet flashCardSet);
```

Comment:
I think we might be returning instance after updating (check with other services)